packages/malloy-render/src/html/duration.ts

```diff
@@ -21,16 +21,95 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-import {DataColumn, Explore, Field} from '@malloydata/malloy';
+import {AtomicField, DataColumn, Explore, Field} from '@malloydata/malloy';
 import {HTMLTextRenderer} from './text';
 import {
   DurationRenderOptions,
   DurationUnit,
+  isDurationUnit,
   StyleDefaults,
 } from '../data_styles';
 import {RendererOptions} from '../renderer_types';
 import {Renderer} from '../renderer';
 import {RendererFactory} from '../renderer_factory';
+import { format } from 'ssf';
+
+export function formatTimeUnit(value: number, unit: DurationUnit, options: { numFormat?: string; terse?: boolean } = {}) {
+  let unitString = unit.toString();
+    if(options.terse) unitString = terseDurationUnitsMap.get(unit) ?? unitString;
+    else if (value === 1) {
```

Comment:
nit: can we add parenthesis here? I feel it would be more readable.

Reply:
done